test/Interop/Cxx/class/Inputs/loadable-types.h

```diff
@@ -0,0 +1,98 @@
+#ifndef TEST_INTEROP_CXX_CLASS_INPUTS_LOADABLE_TYPES_H
+#define TEST_INTEROP_CXX_CLASS_INPUTS_LOADABLE_TYPES_H
+
+
+struct EmptyStruct {};
+
+struct StructWithDefaultConstructor {
+  StructWithDefaultConstructor() {}
+};
+
+struct StructWithAdditionalConstructor {
+  StructWithAdditionalConstructor(){};
+  StructWithAdditionalConstructor(int parameter){};
+};
+
+struct StructWithCopyConstructor {
+  StructWithCopyConstructor(const StructWithCopyConstructor &) {}
+};
+
+struct StructWithInheritedCopyConstructor : StructWithCopyConstructor {};
+
+struct StructWithContainedCopyConstructor {
+  StructWithCopyConstructor member;
+};
+
+struct StructWithMoveConstructor {
+  StructWithMoveConstructor(StructWithMoveConstructor &&) {}
+};
+
+struct StructWithInheritedMoveConstructor : StructWithMoveConstructor {};
+
+struct StructWithContainedMoveConstructor {
+  StructWithMoveConstructor member;
+};
+
+struct StructWithCopyAssignment {
+  StructWithCopyAssignment &operator=(const StructWithCopyAssignment &) {}
+};
+
+struct StructWithInheritedCopyAssignment : StructWithCopyAssignment {};
+
+struct StructWithContainedCopyAssignment {
+  StructWithCopyAssignment member;
+};
+
+struct StructWithMoveAssignment {
+  StructWithMoveAssignment &operator=(StructWithMoveAssignment &&) {}
+};
+
+struct StructWithInheritedMoveAssignment : StructWithMoveAssignment {};
+
+struct StructWithContainedMoveAssignment {
```

Comment:
Sorry, this is a really pedantic nitpick, but could you replace "Contained" with "Subobject" throughout the patch? "subobject" is a term that is used in the C++ spec.

Reply:
Done.